spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1672,46 +1672,45 @@ The container must provide implementation for all these configurators and make t
 
 [[interception_factory]]
 
-=== Apply interceptor programmatically
-
-CDI 2.0 introduces the `javax.enterprise.inject.spi.InterceptionFactory<T>` interface to add services to an instance not created by the container by making each of its method invocation, a business method invocation as defined in <<biz_method>>.
-It will bind business methods only to `@AroundInvoke` interceptor methods.
+=== The `InterceptionFactory` interface
 
+CDI 2.0 introduces the `javax.enterprise.inject.spi.InterceptionFactory<T>` interface which allows to create a wrapper instance whose method invocations are intercepted by method interceptors and forwarded to a provided instance.
```

Comment:
Tiny note; I think there should be comma here:
`... interface, which allows to ...`